ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -114,6 +120,35 @@ def create_tf_graph(self):
         """
         pass
 
+    @staticmethod
+    def _convert_version_string(version_string: str) -> Tuple[int, ...]:
+        """
+        Converts the version string into a Tuple of ints (major_ver, minor_ver, patch_ver).
+        :param version_string: The semantic-versioned version string (X.Y.Z).
+        :return: A Tuple containing (major_ver, minor_ver, patch_ver).
+        """
+        split_ver = version_string.split(".")[0:3]  # Remove dev tag
```

Comment:
I think you can use `distutils.version.LooseVersion` to simplify this:
```
>>> from distutils.version import LooseVersion
>>> v = LooseVersion("1.2.3.dev4")
>>> v
LooseVersion ('1.2.3.dev4')
>>> v.version
[1, 2, 3, 'dev', 4]
>>> v.version[0:3]
[1, 2, 3]
```

Reply:
Oh, that is very handy! Updated to use the LooseVersion class.